src/source/PeerConnection/Rtcp.c

```diff
@@ -20,6 +21,13 @@ STATUS onRtcpPacket(PKvsPeerConnection pKvsPeerConnection, PBYTE pBuff, UINT32 b
         CHK_STATUS(setRtcpPacketFromBytes(pBuff + currentOffset, buffLen - currentOffset, &rtcpPacket));
 
         switch (rtcpPacket.header.packetType) {
+            case RTCP_PACKET_TYPE_FIR:
+                // TODO handle FIR packet https://tools.ietf.org/html/rfc2032#section-5.2.1
+                ssrc1 = getUnalignedInt32BigEndian((rtcpPacket.payload + (SIZEOF(UINT32))));
+                if (STATUS_SUCCESS == findTransceiverBySsrc(pKvsPeerConnection, &pTransceiver, ssrc1)) {
```

Comment:
Minor: There is STATUS_SUCCEEDED macro that you could use like if (STATUS_SUCCEEDED(findTransceiverBySsrc()) {}

This is needed in case if we add other statuses that will be deemed as success later on